### PR TITLE
chore(devenv): use Plex in Storybook manager UI

### PR DESCRIPTION
**Pull request by asudoh**
### Related Ticket(s)

Refs #4423.

### Description

Makes a change to React Storybook environment to use Plex in manager UI. Also fixes an issue in Web Components Storybook where:

* Plex is used only in Chrome
* Plex is not used for code snippet

### Changelog

**Changed**

- A change to React Storybook environment to use Plex in manager UI.
- Fix for an issue in Web Components Storybook where:
  - Plex is used only in Chrome
  - Plex is not used for code snippet

### CODE REVIEW

File: packages/react/.storybook/theme.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+export default create({
+  base: 'light',
+  brandTitle: 'Carbon for IBM.com React',
+  brandUrl:
+    'https://github.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/react',
+  fontBase: '"IBM Plex Sans", "Helvetica Neue", Arial, sans-serif',
+  fontCode:
+    '"IBM Plex Mono", Menlo, "DejaVu Sans Mono", "Bitstream Vera Sans Mono", Courier, monospace',
+});
```
Comment (jeffchew) [inline packages/react/.storybook/theme.js:20]: Sorry, just noticed this after I made a comment in the issue! Thank you for opening this so quickly!

As mentioned, I'm wondering if we should have this centralized in the `storybook-addon-theme` package, so the theme can be shared between react and web components? I think changing other things like colors might be something worth looking into, and would be nice to manage that in one place.

Comment (asudoh) [reply packages/react/.storybook/theme.js:20]: @jeffchew `storybook-addon-theme` is for the UI to choose Carbon color theme, and it doesn't have (or deal with) theme content at all. Sharing Storybook theme content is an interesting idea, though, we can try it once we get more Storybook theme content.

Comment (jeffchew) [reply packages/react/.storybook/theme.js:20]: Thanks, and I think this is probably a future conversation anyway, as discussed we will add this to the design team's ideation board for further research around this.

**Review by jeffchew — APPROVED** [review]
Thank you @asudoh !

**Pull request merged**